client/src/components/Lists/PitchItems/index.tsx

```diff
@@ -0,0 +1,48 @@
+import React, { FC, ReactElement } from 'react';
+import { IPitch } from 'ssw-common';
+
+import { FieldTag } from '../..';
+import { useInterests, useTeams } from '../../../contexts';
+
+interface PitchItemsProps {
+  pitch: Partial<IPitch>;
```

Comment:
Does this need to be a Partial?

Reply:
The pitches (submitted and claimed) I got from IUserAggregate for the user were partial pitches.